### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by thesudhanshupanda**

### CODE REVIEW

File: CSS/Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.alert{
+    width: 100%;
+    padding: .8rem 1.5rem;
+    color: #fff;
+    background-color: var(--primary-color);
+    font-size: 1rem;
+    font-weight: 500;
+    box-shadow: 0 0 3px rgba(0, 0, 0, 0.5);
```
Comment (tanishq20) [inline CSS/Components/alert.css:8]: Here also you can use var for color.

**Review by tanishq20 — APPROVED** [review]
Everything looks good.

File: CSS/Components/alert.css (hunk 1 above)
Comment (thesudhanshupanda) [reply CSS/Components/alert.css:8]: yeah nice catch, will do it

File: CSS/Components/alert.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.alert{
+    width: 100%;
+    padding: .8rem 1.5rem;
+    color: #fff;
+    background-color: var(--primary-color);
```
Comment (shuntzhere) [inline CSS/Components/alert.css:5]: Use variables for colors.

**Review by shuntzhere — APPROVED** [review]
HTML is clean, looks good. Suggested a minor change to be made.

File: CSS/Components/alert.css (hunk 2 above)
Comment (thesudhanshupanda) [reply CSS/Components/alert.css:5]: yeah nice catch, will do it.

**Pull request merged**